spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/sql/TestSelect.java

```diff
@@ -55,6 +56,8 @@ public TestSelect(String catalogName, String implementation, Map<String, String>
   public void createTables() {
     sql("CREATE TABLE %s (id bigint, data string, float float) USING iceberg", tableName);
     sql("INSERT INTO %s VALUES (1, 'a', 1.0), (2, 'b', 2.0), (3, 'c', float('NaN'))", tableName);
+    sql("CREATE TABLE %s (id bigint, binary binary) USING iceberg", binaryTableName);
+    sql("INSERT INTO %s VALUES (1, X''), (2, X'1111'), (3, X'11')", binaryTableName);
```

Comment:
This table is only used in `testBinaryInFilter`. Can you move these to the start of that test instead of creating the table for every test?

You can leave the drop in `removeTables`, though. It doesn't hurt to always attempt to drop.

Reply:
done